app/server/utils/credit_transfer.py

```diff
@@ -257,13 +256,11 @@ def make_payment_transfer(transfer_amount,
             if use_id != 'null':
                 use = TransferUsage.query.get(int(use_id))
                 if use:
-                    usages.append(use.name)
+                    transfer.transfer_usages.append(use)
                     if use.is_cashout:
                         make_cashout_incentive_transaction = True
                 else:
-                    usages.append('Other')
-
-        transfer.transfer_use = usages
+                    raise Exception(f'{use_id} not a valid transfer usage')
```

Comment:
Should we create a generic `other` usage for this case, or is it fair to count on real usages to be used?

Reply:
I think it's fair to count on the IDs being real IDs. That being said, doing this
```
use_ids = transfer_use.split(',')  # passed as '3,4' etc.
```
shouldn't be the role of this function, but rather at the top of an API or similar.

Reply:
@enjeyw Fixed!